test/distributed/_tensor/test_dtensor.py

```diff
@@ -245,6 +245,22 @@ def test_to_local(self):
         except RuntimeError:
             self.assertEqual(sharded_tensor.grad.stride(), [1, 3 * self.world_size])
 
+    @with_comms
+    def test_dtensor_new_empty_strided(self):
+        device_mesh = DeviceMesh(self.device_type, torch.arange(self.world_size))
+        local_tensor = torch.randn(8, 8, requires_grad=True, device=self.device_type)
+        my_dtensor = distribute_tensor(local_tensor, device_mesh, [Shard(0)])
+        new_strided_dtensor = my_dtensor.new_empty_strided((8, 8), (8, 1))
+        self.assertEqual(new_strided_dtensor.shape, my_dtensor.shape)
```

Comment:
Any reason why we don't test BWD?

Reply:
you mean after new_empty_strided then sum().backward? I can write that test too